services/youtube/youtube-subscribers.service.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const { documentation, YouTubeChannelBase } = require('./youtube-channel-base')
+
+module.exports = class YouTubeSubscribes extends YouTubeChannelBase {
+  static route = {
+    base: 'youtube/channel/subscribers',
+    pattern: ':channelId',
+  }
+
+  static get examples() {
+    const preview = this.render({
+      statistics: { subscriberCount: 14577 },
+      channelId: 'UCTNq28Ah5eyDgsYOA0oHzew',
+    })
+    // link[] is not allowed in examples
+    delete preview.link
+    return [
+      {
+        title: 'YouTube Subscriber Counts',
```

Comment:
Could this please be changed to `YouTube Channel Subscribers`? As you'll notice in #6304, the YouTube compliance team was extremely picky about some of the wording used in our UI, I don't want to give them any opportunity to claim that it's not clear that the badge relates to channels.